### PR TITLE
Test (travis) using Julia 1.2 and 1.3 too

### PR DESCRIPTION
**Pull request by lbenet**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,6 +7,8 @@ os:
 julia:
     - 1.0
     - 1.1
```
Comment (ararslan) [inline .travis.yml]: We could probably replace 1.1 with 1.2 since 1.1 has been effectively superseded by 1.2.

Comment (lbenet) [reply .travis.yml]: Thanks for taking a look on this. So, do you suggest to only test Julia for 1.2, 1.3 and nightly?

Comment (ararslan) [reply .travis.yml]: I'd do 1.0 (long term support release), 1.2 (current stable), 1.3 (current unstable), and nightly

Comment (lbenet) [reply .travis.yml]: Thanks for the suggestion; I'll push a new commit following it.

**Comment by lbenet** [comment]
Thanks @ararslan for your review! 

Merging.

**Pull request merged**